src/Theming/theme.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {
+  BLUE_COLORS,
+  BLUE_ENUM,
+  GREEN_COLORS,
+  GREEN_ENUM,
+  NEUTRAL_COLORS,
+  NEUTRAL_ENUM,
+  ORANGE_COLORS,
+  ORANGE_ENUM,
+  PINK_COLORS,
+  PINK_ENUM,
+  PURPLE_COLORS,
+  PURPLE_ENUM,
+  STATUS_COLORS,
+  STATUS_ENUM,
+} from "./colorDefinitions";
+
+export const COLORS = {
+  ...BLUE_ENUM,
+  ...GREEN_ENUM,
+  ...NEUTRAL_ENUM,
+  ...ORANGE_ENUM,
+  ...PINK_ENUM,
+  ...PURPLE_ENUM,
+  ...STATUS_ENUM,
+} as const;
+
+export type ColorVariants = typeof COLORS[keyof typeof COLORS];
+
+export interface Theme {
+  color: Record<ColorVariants, string>;
+}
+
+export const theme: Theme = {
+  color: {
+    ...BLUE_COLORS,
+    ...GREEN_COLORS,
+    ...NEUTRAL_COLORS,
+    ...ORANGE_COLORS,
+    ...PINK_COLORS,
+    ...PURPLE_COLORS,
+    ...STATUS_COLORS,
+  },
+};
+
+export const LUIThemeProvider = ({
```

Comment:
Do you remember @david-gomey if this Provider is useful right now? I thought it was related to having more than one instance of styled-components, but we fixed it by defining it as peer dependency.

Reply:
This Provider is only rendering children, `<>{children}</>`, if we want to create a component that automatically adds the theme perhaps should be `<ThemeProvider theme={theme}>{children}</ThemeProvider>` with ThemeProvider from styled-components.

Reply:
In my opinion, if we want to control from here the theme, perhaps it is useful. If we want to add more flexibility from the app that use the theme, perhaps we only need to expose the theme and the app should add `<ThemeProvider theme={theme}>{children}</ThemeProvider>`

Reply:
Yes, I would remove it too if there's no specific use case for it.